lib/ofdm_synchronization_cvf_impl.cc

```diff
@@ -0,0 +1,286 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2017 Moritz Luca Schmid, Communications Engineering Lab (CEL) / Karlsruhe Institute of Technology (KIT).
+ *
+ * This is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3, or (at your option)
+ * any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include <stdio.h>
+#include <sstream>
+#include <string>
+#include <boost/format.hpp>
+#include <gnuradio/io_signature.h>
+#include "ofdm_synchronization_cvf_impl.h"
+#include <gnuradio/fxpt.h>
+#include <cmath>
```

Comment:
There's a canonical order for includes that people nowadays are striving to attain.

however, I can't even remember it, so I frankly don't care. If you wanted to make things very pretty, you might decide to make a consisten order, such as personal headers first, then GNU Radio headers, then standard headers, or so.